float32[] data
geometry_msgs/Vector3[] points3d
int32 width
int32 height